### ISSUE TITLE
why do i get all zeros when i use roi_align

### ISSUE THREAD

**Issue by zxyyxzz** [facebookresearch/detectron2#2831]
i import torchvision.ops.roi_align

i use the code as below:

`        axis= torch.tensor([count, 0.0, 0.0, 116.0, 116.0],
                                device=left_TopCorner_axis.device)
        roi_obj_Feature = roi_align(embedding, axis, [3,3])` 

the roi_obj_Feature will output all zeros in some embeddings,

**Comment by zxyyxzz** [comment]
This is my code:

`    x = torch.zeros((1,1,117,117), dtype=torch.float)
    x[:,:,53:89,88:102]=100.0
    axis = torch.nonzero(x)
    a, _ = axis.max(dim=0)
    b, _ = axis.min(dim=0)

    rois = torch.tensor([
        # [0,0.0,0.0,0.0,0.0],
        [0.0, 53.0, 89.0, 88.0, 102.0],
    ])
    count = torch.tensor(0).view(1)


    ya = roi_align(x, rois, output_size, sampling_ratio=1)
    print(ya)
`
the out put is :
`float64
tensor([  0,   0,  88, 101])
tensor([ 0,  0, 53, 88])
tensor([[[[0., 0., 0., 0., 0., 0., 0.],
          [0., 0., 0., 0., 0., 0., 0.],
          [0., 0., 0., 0., 0., 0., 0.],
          [0., 0., 0., 0., 0., 0., 0.],
          [0., 0., 0., 0., 0., 0., 0.],
          [0., 0., 0., 0., 0., 0., 0.],
          [0., 0., 0., 0., 0., 0., 0.]]]])
torch.Size([1, 1, 7, 7])`

**Comment by zxyyxzz** [comment]
`    output_size = (7,7)
    spatial_scale = 1/4
    sampling_ratio = 2

    # matrix = np.loadtxt("/home/guopx/zxy/middle_result/for_roi/roi.txt",delimiter=",",skiprows=0)
    # print(matrix.dtype)
    # x=torch.tensor(matrix.reshape(1,1,117,117)).float()

    x = torch.zeros((1,1,117,117), dtype=torch.float)
    x[:,:,53:89,88:102]=100.0
    axis = torch.nonzero(x)
    a, _ = axis.max(dim=0)
    b, _ = axis.min(dim=0)

    rois = torch.tensor([
        # [0,0.0,0.0,0.0,0.0],
        [0.0, 53.0, 89.0, 88.0, 102.0],
    ])
    count = torch.tensor(0).view(1)


    ya = roi_align(x, rois, output_size, sampling_ratio=1)
    print(ya)
`

the out put is:
tensor([[[[0., 0., 0., 0., 0., 0., 0.],
          [0., 0., 0., 0., 0., 0., 0.],
          [0., 0., 0., 0., 0., 0., 0.],
          [0., 0., 0., 0., 0., 0., 0.],
          [0., 0., 0., 0., 0., 0., 0.],
          [0., 0., 0., 0., 0., 0., 0.],
          [0., 0., 0., 0., 0., 0., 0.]]]])

**Comment by zxyyxzz** [comment]
@ppwwyyxx

**Comment by ppwwyyxx** [comment]
You misinterpret x and y.

The code does not involve detectron2, therefore not a detectron2 issue.
